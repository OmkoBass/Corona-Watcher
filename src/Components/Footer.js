import React from 'react'

function Footer() {
    return (
        <footer>
            <p>Corona virus API</p>
            <p>OmkoBass Github</p>
        </footer>
    )
}

export default Footer;
