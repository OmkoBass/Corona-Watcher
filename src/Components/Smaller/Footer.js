import React from 'react'

//Ant Components
import {Layout} from 'antd';

function Footer() {
    return <Layout.Footer style={{display: 'flex', flexDirection: 'column', textAlign: 'center'}}>
        @OmkoBass
        <a href="https://corona.lmao.ninja">CORONA NINJA API</a>
        <a href="https://documenter.getpostman.com/view/8854915/SzS7R6uu?version=latest">NOVEL COVID API</a>
    </Layout.Footer>
}

export default Footer;
