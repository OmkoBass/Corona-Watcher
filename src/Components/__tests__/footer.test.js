import React from "react";

import { render, screen, cleanup } from "@testing-library/react";

import renderer from "react-test-renderer";

import Footer from "../Footer";

afterEach(() => {
  cleanup();
});

it("Should render footer", () => {
  render(<Footer />);

  const footerElement = screen.getByTestId("footer");

  expect(footerElement).toBeInTheDocument();
});

it("Contains p tag with omkobass", () => {
  render(<Footer />);

  const footerElement = screen.getByTestId("footer");

  expect(footerElement).toContainHTML("<p>OmkoBass Github</p>");
});

it("Renders correctly", () => {
  const tree = renderer.create(<Footer />).toJSON();

  expect(tree).toMatchSnapshot();
});
