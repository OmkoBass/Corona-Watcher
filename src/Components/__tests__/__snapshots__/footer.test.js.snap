// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<footer
  className="p-3"
  data-testid="footer"
>
  <a
    href="https://github.com/NovelCOVID/API"
  >
    https://github.com/NovelCOVID/API
  </a>
  <p>
    OmkoBass Github
  </p>
</footer>
`;
